client/assets/js/board.js

```diff
@@ -41,6 +40,16 @@ export default class Board {
 			board.snakes[0].move();
 		}, DELAY);
 
+		board.snakes.forEach(function(e){
```

Comment:
There should not be direct DOM manipulation inside board.js. Please create a separate file (like `updateScoreBoard` or something)